### DISCUSSION TITLE
Does homebrew still symlink files to /usr/local/bin on Big Sur?

### DISCUSSION THREAD

**Discussion by reedrosenbluth** [Homebrew/discussions#1024] [Everyday usage]
Since installing Homebrew on Big Sur, I've noticed that everything gets installed in `/opt/homebrew/Cellar` and that nothing is getting symlinked to `/usr/local/bin`. Is this correct behavior?

**Comment by carlocab** [comment]
On Intel Big Sur, yes. On ARM, no. There, default prefix is `/opt/homebrew`, and formulae are symlinked into `/opt/homebrew/{bin,lib,include}`, etc.

**Comment by SMillerDev** [comment]
On M1 devices the homebrew prefix is `/opt/Homebrew`. That's correct.
